File: js/management-api-v2/DeleteType.js
// DocSection: cm_api_v2_delete_type
// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
import { ManagementClient } from '@kentico/kontent-management';

const client = new ManagementClient({
  projectId: '<YOUR_PROJECT_ID>',
  apiKey: '<YOUR_API_KEY>'
});

const response = await client.deleteContentType()
  .byTypeId('269202ad-1d9d-47fd-b3e8-bdb05b3e3cf0')
  // .byTypeCodename('hosted_video')
  .toPromise();
// EndDocSection

File: net/management-api-v2/DeleteType.cs
// DocSection: cm_api_v2_delete_type
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_PROJECT_ID>"
});

var identifier = Reference.ById(Guid.Parse("269202ad-1d9d-47fd-b3e8-bdb05b3e3cf0"));
// var identifier = Reference.ByCodename("hosted_video");
// var identifier = Reference.ByExternalId("Content-Type-123");

await client.DeleteContentTypeAsync(identifier);
// EndDocSection